.env.example

```
@@ -0,0 +1 @@
+VITE_MAPBOX_TOKEN=XXX
```

Add `"files.insertFinalNewline": true,` to your IDE settings 🙂 


```suggestion
VITE_MAPBOX_TOKEN=XXX

```